safe_core/src/client/mock/vault.rs

```diff
@@ -934,13 +927,13 @@ impl Vault {
                     .get_adata(append.address, requester_pk, request)
                     .and_then(move |data| match data {
                         AData::PubSeq(mut adata) => {
-                            unwrap!(adata.append(&append.values, index));
+                            unwrap!(adata.append(append.values, index));
```

Comment:
This can be changed to
```suggestion
                            adata.append(append.values, index)?;
```
There are a few more occurrences. Could you please change them